dice-game-ui-angular/src/app/start-game/start-game.component.ts

```diff
@@ -1,17 +1,30 @@
 import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { SimpleGameEngine } from '../../../../engine/simple-game-engine';
```

Comment:
You don't need to do this. Due to changes in tsconfig.json file you can now use import from simple `'engine'` module.

Reply:
Fixed